abb_irb120_moveit_config/launch/trajectory_execution.launch.xml

```diff
@@ -7,9 +7,11 @@
   <param name="moveit_manage_controllers" value="$(arg moveit_manage_controllers)"/>
 
   <!-- When determining the expected duration of a trajectory, this multiplicative factor is applied to get the allowed duration of execution -->
-  <param name="allowed_execution_duration_scaling" value="1.2"/> <!-- default 1.2 -->
+  <param name="trajectory_execution/allowed_execution_duration_scaling" value="1.2"/> <!-- default 1.2 -->
   <!-- Allow more than the expected execution time before triggering a trajectory cancel (applied after scaling) -->
-  <param name="allowed_goal_duration_margin" value="0.5"/> <!-- default 0.5 -->
+  <param name="trajectory_execution/allowed_goal_duration_margin" value="0.5"/> <!-- default 0.5 -->
+  <!-- Disable execution duration monitoring to prevent trajectory cancel -->
+  <param name="trajectory_execution/execution_duration_monitoring" value="false" /> <!-- default false -->
```

Comment:
O. Hm. That's a more debatable one.

I'd say no, but the current driver is not performant enough, so the TEM will cancel allmost all traj executions. So it might make sense to disable it by default.

*However*: the title of this PR is "namespace change", and as such it should not introduce such a change and disabling the TEM in the first place.

@tycho94: could you please only fix the TEM parameter namespace and not include the TEM disable change in this PR? If you'd like to discuss whether we should disable it by default, please open an issue first, so we can properly weight the pros and cons.